src/components/interpretation-modal/interpretation-modal.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from 'react'
+import {
+    useInterpretationModalState,
+    useInterpretationModalTogglers,
+} from '@components/app-wrapper/interpretations-provider'
+import type { MetadataInput } from '@components/app-wrapper/metadata-helpers'
+import { ModalDownloadDropdown } from '@components/download-menu/modal-download-dropdown'
+import { PluginWrapper } from '@components/plugin-wrapper/plugin-wrapper'
+import { InterpretationModal as AnalyticsInterpretationModal } from '@dhis2/analytics'
+import { useAppSelector } from '@hooks'
+import { getIsVisualizationLoading } from '@store/loader-slice'
+import { getSavedVis } from '@store/saved-vis-slice'
+
+export const InterpretationModal: FC = () => {
+    const savedVis = useAppSelector(getSavedVis)
+    const { interpretationId, initialFocus } = useInterpretationModalState()
+    const { onCloseInterpretationModal } = useInterpretationModalTogglers()
+    const isVisualizationLoading = useAppSelector(getIsVisualizationLoading)
+
+    // No need to store metadata from the interpretaion modal analytics response
+    // The metadata is not accessed by the plugin anyway
+    const noop: (metadata: MetadataInput) => void = () => {}
```

Comment:
I see why we need to do this: `onResponseReceived` is a required prop in the `InterpretationModal` in analytics... So this makes sense. But perhaps it's better to simply declare the `noop` in the module scope so we don't run the risk to get an unnecessary unstable reference.

And perhaps clarify with a comment that this callback is a required prop.

Reply:
Thanks for the heads up.
Fixed.